lib/web/appserver.rb

```diff
@@ -519,7 +525,9 @@ def start_device_ejectable_event
           download: %!<a href="/novels/#{id}/download" class="btn btn-default btn-xs"><span class="glyphicon glyphicon-download-alt"></span></a>!,
           frozen: Narou.novel_frozen?(id),
           new_arrivals_date: data["new_arrivals_date"].tap { |m| break m.to_i if m },
-          general_lastup: data["general_lastup"].tap { |m| break m.to_i if m }
+          general_lastup: data["general_lastup"].tap { |m| break m.to_i if m },
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'm' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->